src/wktplot/plots/osm.py

```diff
@@ -2,18 +2,32 @@
 from bokeh.tile_providers import Vendors, get_provider
 from typing import Any, Dict
 from wktplot.mappers.osm import OpenStreetMapper
+from wktplot.mappers.standard import StandardMapper
 from wktplot.plots.standard import WKTPlot
 
 
 class OpenStreetMapsPlot(WKTPlot):
     """ OpenStreetMaps WKTPlot Bokeh wrapper class.
     """
 
-    mapper = OpenStreetMapper
-    default_figure_style_kwargs: Dict[str, str] = {
-        "x_axis_type": "mercator",
-        "y_axis_type": "mercator",
-    }
+    def __init__(self, *args, disable_mercator: bool = False, **kwargs) -> None:
+        """ Create figure with given arguments.
+
+        Args:
+            title (str): Title for graph and output filename, defaults to random string if unset.
+            save_dir (str | obj: Path | None, default = None): Optional path to save output file to.
+            disable_mercator (bool, default=False): Disable mercator proction calculation for shape data.
+            **figure_style_kwargs (dict[str, Any]): Dictionary of attributes to style the created figure.
+                See this guide for available style attributes:
+                https://docs.bokeh.org/en/2.4.3/docs/reference/plotting/figure.html
+
+        Raises:
+            ValueError: If value for `title` is not a string or None.
+            OSError: If value for `save_dir` is not a directory.
+        """
+
+        super().__init__(*args, **kwargs)
+        self.mapper = StandardMapper if disable_mercator else OpenStreetMapper
```

Comment:
Should we rename these mapper classes to reference their projection? `StandardMapper` -> `GeometricMapper` and `OpenStreetMapper` -> `MercatorMapper`.

Reply:
Yes, I'll certainly find that helpful in the future when I forget which is "Standard"

Reply:
Ok, will do in a followup PR.